neo/Ledger/RelayResultReason.cs

```diff
@@ -2,12 +2,12 @@
 {
     public enum RelayResultReason : byte
     {
-        Succeed,
-        AlreadyExists,
-        OutOfMemory,
-        UnableToVerify,
-        Invalid,
-        PolicyFail,
-        Unknown
+        Succeed = 0x00,
```

Comment:
Do we really need explicit enumeration values?

Reply:
Perhaps the idea was to avoid changing the value of Unknown again, since Unknown was changed to PolicyFail. Maybe only needs to be explicit for unknown.

Reply:
I think explicit values make it easier for documentation purposes, otherwise the developer will need to count one by one just to know the expected value xD Regarding Unknown, it's good to have it fixed, even with future changes. If you prefer other numbers, we can change too.

Reply:
For documentation, we can just use the enumeration names, such as `Succeed`, `AlreadyExists`, etc.
The enumeration values are not used anywhere. Even RpcServer is using enumeration names.

Reply:
I'll rethink this, because I'm probably very wrong then :)

Reply:
Ok, now I got your point Erik, I'm actually wrong. But still, perhaps a better idea is to automate the Rpc result numbers using -500 + enum value. So, as soon as we add more, it will get automatically documented. Do you think that's reasonable? Otherwise, we can just remove my commit.

Reply:
I think I took for granted that they were not actually exposed outside anywhere. Nice it wasn’t needed.

Reply:
I tried, but it was a bad idea... too complex because of the subtraction, it would get more complicated, what is not good. Sorry for the misunderstanding.